model/riscv_insts_aext.sail

```diff
@@ -86,14 +86,14 @@ function clause execute(LOADRES(aq, rl, rs1, width, rd)) = {
       /* "LR faults like a normal load, even though it's in the AMO major opcode space."
         * - Andrew Waterman, isa-dev, 10 Jul 2018.
         */
-      if not(is_aligned(vaddr, width))
-      then { handle_mem_exception(vaddr, E_Load_Addr_Align()); RETIRE_FAIL }
+      if not(is_aligned(virtaddr_bits(vaddr), width))
+      then { handle_mem_exception(virtaddr_bits(vaddr), E_Load_Addr_Align()); RETIRE_FAIL }
```

Comment:
I think handle_mem_exception always takes a virtual address, so maybe you can change its parameter to virtaddr?

Reply:
Thank you for pointing that out, so I won't be tormented by multiple calls in vext_mem, I will fix it.